src/scss/grommet-core/_objects.table.scss

```diff
@@ -1,5 +1,34 @@
 // (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
 
+@mixin small-table {
+
+  thead {
+    display: none;
+  }
+
+  td {
+    display: block;
+
+    &:before {
+      font-weight: 100;
+      @include inuit-font-size(18px);
```

Comment:
Why 18px? Can we tie this to a font related variable we already have, such as $control-font-size?